CMakeLists.txt

```diff
@@ -791,26 +803,26 @@ function(add_compiler_rt_tests variant)
     add_dependencies(check-llvm-toolchain-runtimes-${variant} check-compiler-rt-${variant})
 endfunction()
 
-function(add_libcxx_libcxxabi_libunwind_tests variant)
+function(add_libcxx_libcxxabi_libunwind_tests variant abi_version rtti)
```

Comment:
It would be more consistent to have same new parameters as in `add_libcxx_libcxxabi_libunwind`: `enable_exceptions` `enable_rtti`. Extending the target name could be then extracted to separate function: `extend_target_name(target_name ${enable_exceptions} ${enable_rtti})`